doc/user/content/get-started.md

```diff
@@ -5,216 +5,271 @@ menu: "main"
-1. From your Materialize CLI, create a materialized view that contains actual data we can work with.
+1. You can then connect to the running instance using any [Materialize-compatible CLI](/connect/cli/), like `psql` or `mzcli`. If you have `psql` installed on your machine, connect using:
 
-    ```sql
-    CREATE MATERIALIZED VIEW pseudo_source (key, value) AS
-        VALUES ('a', 1), ('a', 2), ('a', 3), ('a', 4),
-        ('b', 5), ('c', 6), ('c', 7);
+    ```shell
+    psql -U materialize -h localhost -p 6875 materialize
     ```
 
-    You'll notice that we end up entering data into Materialize by creating a materialized view from some other data, rather than the typical `INSERT` operation. This is how one interacts with Materialize. In most cases, this data would have come from an external source and get fed into Materialize from a file or a stream.
+    Otherwise, you can find the steps to install and use your CLI of choice under [Install](/install/#cli-connections).
 
-1. With data in a materialized view, we can perform arbitrary `SELECT` statements on the data.
+{{< /tab >}}
 
-    Let's start by viewing all of the data:
+{{< tab "Linux">}}
 
-    ```sql
-    SELECT * FROM pseudo_source;
-    ```
-    ```nofmt
-     key | value
-    -----+-------
-     a   |     1
-     a   |     2
-     a   |     3
-     a   |     4
-     b   |     5
-     c   |     6
-     c   |     7
+1. Assuming you're using [apt](https://linuxize.com/post/how-to-use-apt-command/), open a terminal and run (as _root_):
+
+    ```shell
+    # Add the signing key for the Materialize apt repository
+    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 79DEC5E1B7AE7694
+    # Add and update the repository
+    sh -c 'echo "deb http://apt.materialize.com/ generic main" > /etc/apt/sources.list.d/materialize.list'
+    apt update
+    # Install materialized
+    apt install materialized
     ```
 
-1. Determine the sum of the values for each key:
+    **Note:** For a _curl_-based alternative, see [Install](/install/#curl-1).
 
-    ```sql
-    SELECT key, sum(value) FROM pseudo_source GROUP BY key;
+1. Once the installation is complete, you can start the `materialized` process:
+
+    ```shell
+    materialized -w 1
     ```
-    ```nofmt
-     key | sum
-    -----+-----
-     a   |  10
-     b   |   5
-     c   |  13
+
+    This starts a process using one [worker thread](/connect/cli//#worker-threads) and listening on port 6875 by default.
+
+1. You can then connect to the running instance using any [Materialize-compatible CLI](/connect/cli/), like `psql` or `mzcli`. If you have `psql` installed on your machine, connect using:
+
+    ```shell
+    psql -U materialize -h localhost -p 6875 materialize
     ```
 
-    We can actually then save this query as its own materialized view:
+    Otherwise, you can find the steps to install and use your CLI of choice under [Install](/install/#cli-connections).
+
+{{< /tab >}}
+
+{{< /tabs >}}
+
+## Explore a streaming source
```

Comment:
## Stream data

Reply:
I might be biased here, but the main purpose of Materialize is not to stream data, but to consume and transform streams of data. So "Stream data" wouldn't be correct here, IMO.